subprojects/docs/build.gradle

```diff
@@ -549,7 +549,7 @@ tasks.named("checkAsciidoctorSampleContents") {
 }
 
 // exclude (unused and non-existing) wrapper of development Gradle version, as well as README, because the timestamp in the Gradle version break the cache
-tasks.withType(InstallSample) {
+tasks.withType(InstallSample).configureEach {
```

Comment:
That may avoid creating the tasks, though I'd still like to know why we have so many tasks in the first place.

Reply:
Because current implementation creates a `InstallSample` task for each `docs/src/snippets` directory: https://github.com/gradle/gradle/blob/bc952afc4cfd14bcda8a46f4161888eefd057d06/subprojects/docs/build.gradle#L199

It seems not simple to get rid of so many tasks, so let's make them lazy for now.

Reply:
I understand. What I don't understand is why we need one task for each sample and not have one task for all samples.